README.md

```diff
@@ -1,29 +1,60 @@
-# Math-Magician
+# 📖 [Bookstore] <a name="Bookstore: connect to API"></a>
 
-This is a book store app build by react library.
+> n this project, I connected my React-Redux application to an existing API to create and remove books in a remote server.
 
-![SCREENSHOT](./src/component/images/)
+**[Bookstore]** connect to API
 
-## Built With
+## 🛠 Built With <a name="built-with"></a>
 
-- ReactJS
+### Tech Stack <a name="tech-stack"></a>
 
-## Video Walkthrough
+> HTML, CSS, JS Webpack, react-redux.
 
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
+<!-- Features -->
 
-## Get Live Demo
+### Key Features <a name="Conneting to api server"></a>
 
-[Live Demo Netlify Link]()
+> Post and get data from api server.
 
+- **[Post]**
+- **[Get]**
+- **[Store]**
 
-## Clone and review it
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Run the following commands to clone and run it.
+<!-- LIVE DEMO -->
 
-### To clone the repository
+## 🚀 Live Demo <a name="live-demo"></a>
 
-  `git clone https://github.com/eerapheal/my-math-apps.git`
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
[ OPTIONAL ] I think it would be better if you update your live demo link with a working one to make your project documentation more professional.